src/daft-core/src/python/series.rs

```diff
@@ -326,12 +336,27 @@ impl PySeries {
                 "ngram_size must be positive: {ngram_size}"
             )));
         }
-        let cast_seed = seed as u32;
+        let seed = seed as u32;
 
-        Ok(self
-            .series
-            .minhash(num_hashes as usize, ngram_size as usize, cast_seed)?
-            .into())
+        let num_hashes = num_hashes as usize;
+        let ngram_size = ngram_size as usize;
+
+        let result = match hash_function {
+            HashFunctionKind::MurmurHash3 => {
+                let hasher = MurBuildHasher::new(seed);
```

Comment:
I believe the seed should be just used for the permutations calculation and not for the hash function but let's double check that.

Reply:
in old code I am 93% sure that seed is used in both places. I agree it is jank... but do we want to pass _two_ seeds in now? Could be add. I suppose hash functions could also be classes or something that could have optional parameters in initializing them that could include seed🤷